packages/table/src/interactions/resizable.tsx

```diff
@@ -97,12 +97,15 @@ export class Resizable extends React.PureComponent<IResizableProps, IResizeableS
         };
     }
 
-    public componentWillReceiveProps(nextProps: IResizableProps) {
-        const { size } = nextProps;
-        this.setState({
-            size,
-            unclampedSize: size,
-        });
+    public componentDidUpdate(prevProps: IResizableProps) {
+        const didSizePropChange = prevProps.size !== this.props.size;
+        const { size } = this.props;
+        if (didSizePropChange) {
```

Comment:
same here, curious why the check is necessary. also if it is, it doesn't need to be stored in a variable, just write `prevProps.size !== size` inline